build-system/tasks/get-zindex/index.js

```diff
@@ -29,6 +29,8 @@ const {getStdout} = require('../../common/process');
 const {gray, magenta} = require('kleur/colors');
 const {logOnSameLineLocalDev, logLocalDev} = require('../../common/logging');
 const {writeDiffOrFail} = require('../../common/diff');
+/** @type {Postcss.default} */
+const postcss = /** @type {*} */ (Postcss);
```

Comment:
nit: Insert a blank line between this block and the lines above. (Helps our linter to auto-sort `require` statements.)